src/app/Models/Filter.ts

```diff
@@ -1,9 +1,9 @@
 export class Filter {
```

Comment:
It's not good name for class. It's not filter for all pages but only for doctors.